### ISSUE TITLE
Jasmine htmlreporter throw error on rendered index.html.

### ISSUE THREAD

**Issue by jakeharding** [jakeharding/django-jasmine#20]
This maybe caused by not loading a default boot.js.  Check jasmine docs.

**Comment by DanielSwain** [comment]
Could be, but the Jasmine code looks like it attempts to provide for a blank dictionary if no environment options are specified:
env = options.env || {}

Seems like if the above was changed to the following, then the problem would go away:
env = "env" in options : options.env ? {}

**Comment by DanielSwain** [comment]
I submitted a pull request to the Jasmine project with the above proposed fix.

**Comment by DanielSwain** [comment]
Not sure why, but my pull request for the above failed the CI.  However, **I MIGHT have found the error** - in lines 51 through 54 of base.html, if you change it to this, then the error goes away:

    var jasmineEnv = jasmine.getEnv();
    jasmineEnv.updateInterval = 1000;

    var htmlReporter = new jasmine.HtmlReporter(jasmineEnv);

Is that the correct fix to the problem?

**Comment by jakeharding** [comment]
Yeah, it makes sense to me and a pretty simple fix.  Feel free to submit a pull request with this.  I should be able to get to it tomorrow.  Sorry I haven't been keeping up with you. I'm on vacation and have some family in town, but I will do my best.  Thanks for all your input!
